Makefile.common

```diff
@@ -107,10 +107,10 @@ endif
 %: common-% ;
 
 .PHONY: common-all
-common-all: precheck style check_license lint yamllint unused build test
+common-all: precheck style check_license lint yamllint unused build test ## Run all Prometheus quality checks and build
```

Comment:
For long lines I would suggest creating extra line like here:
 https://github.com/thanos-io/thanos/blob/main/Makefile#L228C1-L228C52
```suggestion
common-all:  ## Run all Prometheus quality checks and build.
common-all: precheck style check_license lint yamllint unused build test
```